pkgs/development/libraries/libpsl/default.nix

```diff
@@ -24,7 +24,7 @@ stdenv.mkDerivation rec {
   };
 
   # bin/psl-make-dafsa brings a large runtime closure through python3
-  outputs = [ "bin" "out" "dev" ];
+  outputs = lib.optional (!stdenv.hostPlatform.isStatic) "bin" ++ [ "out" "dev" ];
```

Comment:
This is not great as the default output for static is now a different one.

Reply:
Yes, but this way avoids rebuilds. Send a PR for the right change, as you see it?

Reply:
#336884